static/app/views/replays/utils/mergeBreadcrumbsEntries.tsx

```diff
@@ -4,11 +4,34 @@ function isBreadcrumbs(entry: Entry) {
   return entry.type === EntryType.BREADCRUMBS;
 }
 
-export default function mergeBreadcrumbsEntries(events: Event[]): Entry {
+function eventBreadcrumbEntries(event: Event) {
+  return event.entries.filter(isBreadcrumbs);
+}
+function entryValues(entry: Entry) {
+  return entry.data.values;
+}
+
+type FilterPredicate = Parameters<typeof Array.prototype.filter>[0];
+function not(predicate: FilterPredicate): FilterPredicate {
+  return (value, index, array) => !predicate(value, index, array);
+}
```

Comment:
This also feels a bit weird considering we're only using it once?

Reply:
After all that said in the comment aboce... this is the less useful of the bunch since I could invert the logic inside `isRRWebTransaction` directly. Which I can do first.